db/migrate/20181101134526_drop_old_tables.rb

```diff
@@ -0,0 +1,9 @@
+class DropOldTables < ActiveRecord::Migration[5.1]
+  def change
+    drop_table('catalog_items')
+    drop_table('catalog_plans')
+    drop_table('parameter_values')
+    drop_table('plan_parameters')
+    drop_table('providers')
```

Comment:
Do we need to also remove `catalog_parameter` and `json_schema`?

Reply:
@syncrou I am not seeing those tables, in the migration

Reply:
I see now that `ApplicationRecord` is an abstract table ( which `JsonSchema` and `CatalogParameter` inherit from.